schutzbot/run_image_tests.sh

```diff
@@ -62,7 +62,7 @@ run_test_case () {
 
     # Run the test and add the test name to the list of passed or failed
     # tests depending on the result.
-    if sudo $TEST_CMD 2>&1 | tee ${WORKSPACE}/${TEST_NAME}.log; then
+    if sudo --preserve-env $TEST_CMD 2>&1 | tee ${WORKSPACE}/${TEST_NAME}.log; then
```

Comment:
Is this necessary because of the additional environment variables that are read above? Why wasn't it necessary for the other vars?

Reply:
Ah I see. Thanks!